src/servers/ZoneServer2016/entities/character.ts

```diff
@@ -449,6 +450,19 @@ export class Character2016 extends BaseFullCharacter {
         server.sendHudIndicators(client);
       }
     }
+    const indexFoodPoison =
+      this.resourceHudIndicators.indexOf("FOOD POISONING");
```

Comment:
use .includes()

Reply:
Messed with this and wasn't able to get the index of the current position of the food poisoning without having to put in some wonky code for it. Also all other resourceHudIndicator calls are using indexOf too.